test/Elastic.Apm.AspNetCore.Tests/MicrosoftExtensionsConfigTests.cs

```diff
@@ -74,11 +76,13 @@ public void ReadInvalidLogLevelConfigFromAppsettingsJson()
 		public void ReadInvalidServerUrlsConfigFromAppsettingsJson()
 		{
 			var logger = new TestLogger();
-			var config = new MicrosoftExtensionsConfig(GetConfig($"TestConfigs{Path.DirectorySeparatorChar}appsettings_invalid.json"), logger);
+			var config = new MicrosoftExtensionsConfig(GetConfig($"TestConfigs{Path.DirectorySeparatorChar}appsettings_invalid.json"), logger,
+				"test");
```

Comment:
Please see my comment above regarding adding an `internal` constructor overload to `MicrosoftExtensionsConfig`.